### ISSUE TITLE
Not able to get Rocket from onRocketLaunched

### ISSUE THREAD

**Issue by recon88** [OxideMod/Oxide#606]
A while ago I was able to use TimedExplosive in the said event to modify Rocket radius and damage.
Somehow that is no longer possible as it returns null. Is this intended or a bug?
I would like to modify Rocket damange only, not explosive damage overall.  

```
void OnRocketLaunched(BasePlayer player, BaseEntity entity)
			var explosive = entity as TimedExplosive;
			if (!explosive) return;
}
```

**Comment by lukespragg** [comment]
Nothing has changed with the hook aside from removing the wrapper, but it may have changed with the game. I see the 2nd parameter is being "BaseLauncher", which is based on "BaseEntity", but that shouldn't affect it.

**Comment by recon88** [comment]
Any workarounds? That event is pretty much useless if you can't modify the rocket itself which was possible before.

**Comment by lukespragg** [comment]
I don't see that it changed, so I'm not sure. I'm not really familiar with the hook personally.

**Comment by Mughisi** [comment]
This was fixed in 84ac330
